src/Controls/src/Core/Platform/iOS/Extensions/BrushExtensions.cs

```diff
@@ -251,6 +251,31 @@ static NSNumber[] GetCAGradientLayerLocations(List<GradientStop> gradientStops)
 			}
 		}
 
+		static CGColor[] GetCAGradientLayerColors(List<GradientStop> gradientStops)
```

Comment:
Are mostly pending changes from https://github.com/xamarin/Xamarin.Forms/pull/13401

Reply:
Should this rather be a reference to the code in Core? Instead of a duplicate? Might be possible if the method was internal.